content/streaming-api/index.textile

```diff
@@ -12,7 +12,9 @@ jump_to:
     - Server-sent events#sse
 ---
 
-The Ably raw HTTP streaming API provides a way to get a realtime stream of events from Ably in circumstances where using a full Ably Realtime client library, or even an MQTT library, is impractical.
+The Ably SSE + raw HTTP streaming API provides a way to get a realtime stream of events from Ably in circumstances where using a full Ably Realtime client library, or even an MQTT library, is impractical.
+
+HTTP streaming allows for a request from a client to be held by a server, allowing it to push data to the client without further requests. This, much like WebSockets, help avoid the overhead involved in normal HTTP requests. "Server-sent events":https://developer.mozilla.org/en-US/docs/Web/API/Server-sent_events (SSE) is another method of achieving this for web pages, usually done through the use of "EventSource":https://developer.mozilla.org/en-US/docs/Web/API/EventSource.
```

Comment:
> Server-sent events (SSE) is another method of achieving this for web pages

@tomasmcm It's not really a different method, it's a thin layer on top of HTTP streaming. It's not limited to web pages either, eg we have SSE python and elixir examples in the tutorials repo